test/playwright/previews.test.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable no-unused-vars */
+import {test, expect} from '@playwright/test'
+import {componentPreviews} from './helpers'
+
+test.beforeEach(async ({page}, testInfo) => {
+  testInfo.snapshotSuffix = ''
```

Comment:
Have you had any trouble with the snapshots on macOS and ubuntu with this change? Would definitely simplify stuff if we didn't have to use docker 👀

Reply:
I wasn't worried about it too much, in a codespace it's the same environment as CI. And if someone did run it on macos and checked it in, the ci would override them with new screenshots since we check in any changes the ci job detects.